components/elm/src/cpl/lnd_comp_mct.F90

```diff
@@ -44,6 +44,7 @@ subroutine lnd_init_mct( EClock, cdata_l, x2l_l, l2x_l, NLFilename )
     use clm_time_manager , only : get_nstep, get_step_size, set_timemgr_init, set_nextsw_cday
     use elm_initializeMod, only : initialize1, initialize2, initialize3
     use elm_instMod      , only : lnd2atm_vars, lnd2glc_vars
+    use elm_instance     , only : elm_instance_init
```

Comment:
I could not find `elm_instance`. Did you forget to commit `elm_instance.F90`?

Reply:
@wlin7 ⬆️

Reply:
Thanks a lot for the careful review, @bishtgautam . I was scratching my head when the auto-tester failed. The file is on my local branch, but was missed when staging the modified files. It is now committed to the branch.